ansible_collections/netapp/ontap/plugins/modules/na_ontap_command.py

```diff
@@ -241,13 +241,13 @@ def parse_xml_to_dict(self, xmldata):
                     if len(stripped_line) > 1:
                         self.result_dict['stdout_lines'].append(stripped_line)
 
-                    # Generate stdout_lines_filter_list
-                    if self.exclude_lines:
-                        if len(stripped_line) > 1 and self.include_lines in stripped_line and self.exclude_lines not in stripped_line:
-                            self.result_dict['stdout_lines_filter'].append(stripped_line)
-                    else:
-                        if len(stripped_line) > 1 and self.include_lines in stripped_line:
-                            self.result_dict['stdout_lines_filter'].append(stripped_line)
+                        # Generate stdout_lines_filter_list
+                        if self.exclude_lines:
+                            if self.include_lines in stripped_line and self.exclude_lines not in stripped_line:
+                                self.result_dict['stdout_lines_filter'].append(stripped_line)
+                        else:
+                            if self.include_lines and self.include_lines in stripped_line:
+                                self.result_dict['stdout_lines_filter'].append(stripped_line)
```

Comment:
Though I asked for this, I now wonder if it can be slightly confusing,
Or maybe we need a note in the documentation to indicate that self.result_dict['stdout_lines_filter'] is empty if no filter is specified.

Reply:
@lonico agree, let's add it to module documentation

Reply:
@zeten30  Are you updating the documentation section?

Reply:
@lonico sorry for delay - https://github.com/ansible/ansible_collections_netapp/pull/11/commits/8223a44382d47bdc53aa21013b4592ac861e2d5c